### PR TITLE
Cluster Autoscaler Packages schema.json

### PR DESCRIPTION
**Pull request by jonathanmeier5**
*Issue #, if available:*

*Description of changes:*
Adding schema to publish helm chart as part of packages bundle.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/kubernetes/autoscaler/1-23/helm/schema.json [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+  ],
+  "extraVolumeMounts": [
+
+  ],
+  "fullnameOverride": "",
+  "image": {
+    "sourceRegistry": "public.ecr.aws",
+    "repository": "kubernetes/autoscaler",
+    "digest": "sha256:372064f0f0ef50ee025109fff55683e50d488b8ae5fe1cb62a7bbc0fc16a64ee",
```
Comment (jonathanmeier5) [inline projects/kubernetes/autoscaler/1-23/helm/schema.json:66]: I'm guessing I should pull out this specific digest, and that the packages controller will populate it with the most current version?

File: projects/kubernetes/autoscaler/1-23/helm/schema.json [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  "clusterAPIMode": "incluster-incluster",
+  "clusterAPIKubeconfigSecret": "",
+  "clusterAPIWorkloadKubeconfigPath": "/etc/kubernetes/value",
+  "clusterAPICloudConfigPath": "/etc/kubernetes/mgmt-kubeconfig",
+  "clusterAPIConfigMapsNamespace": "",
+  "cloudConfigPath": "",
+  "cloudProvider": "clusterapi",
+  "containerSecurityContext": {
+  },
+  "deployment": {
```
Comment (jonathanmeier5) [inline projects/kubernetes/autoscaler/1-23/helm/schema.json:32]: For now I'm leaving all of these fields exposed while I test if that's ok. I expect that I will be able to strip down a lot of them before release.

That said, it seems to me we would rather err on the permissive than restrictive side. Does that align with packages team experience?

Comment (TerryHowe) [reply projects/kubernetes/autoscaler/1-23/helm/schema.json]: yes

File: projects/kubernetes/autoscaler/1-23/helm/schema.json (hunk 1 above)
Comment (TerryHowe) [reply projects/kubernetes/autoscaler/1-23/helm/schema.json]: We'll have a specific schema.json per chart, but yeh, we shouldn't be validating a particular digest in there here

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Comment by jonathanmeier5** [comment]
Merging with @TerryHowe 's approval.

**Pull request merged**